setup.py

```diff
@@ -0,0 +1,38 @@
+from setuptools import setup, find_packages
+from os.path import curdir, realpath
+
+description = "Python library for automation tests of smart cards using "\
+    "virtualization."
+here = realpath(curdir)
+with open(f"{here}/requirements.txt", "r") as f:
+    reqs = f.readlines()
+
+with open(f"{here}/README.md", "r") as f:
+    long_description = f.read()
```

Comment:
Instead of using curdir and then realpath with the value from curdir, you can just use os.getcwd().
Or define variables such as readme = os.getcwd() + '/README.md' and the open(readme)

Reply:
I don't think this is a good option (neither mine). The reason for that is that `os.getcwd()` returns the path from where the file was called. that means if you would call a file that located _/dir1/dir2/dir3/file.py_ from _/dir1/dir2/_ like `python3 dir3/file.py`, `os.getcwd()` would return _/dir1/dir2/_ and it is not very robust. We want to have an absolute path to the file independently in the place where the file was called. Will fix this thing.